### ISSUE TITLE
Why refer only to the 1789 Declaration of the Rights of Man and of the Citizen?

### ISSUE THREAD

**Issue by JulesEllepi** [Staphylococcus/federal-eu-constitution#13]
I see that you refer to the Declaration of the Rights of Man and of the Citizen of 1789: I recognise its importance, but why confine yourself to this source? Why not use sources from the (attempted) revolutions of other European countries? Or from existing national constitutions? Don't get me wrong, I love the French Revolution and am eternally grateful to those revolutionaries - I practically became a pro-European because I wanted to be a fellow citizen of Robespierre and Saint-Just (I mean, who wouldn't?) - but I'm afraid that focusing mainly on France might overshadow the other European nations. Besides, why not refer to the Universal Declaration of Human Rights, which is more modern?

**Comment by Mathew-Iron** [comment]
@JulesEllepi  Yes, that's right, we should take a look and maybe change the 1789 inclusion to something more modern.

**Comment by Mathew-Iron** [comment]
@JulesEllepi By the way, If you have an idea for another document of this type that we should include, please let me know.

**Comment by Staphylococcus** [comment]
I think I agree with @JulesEllepi that the Universal Declaration of Human Rights sounds like good up to date suggestion.

**Comment by Mathew-Iron** [comment]
@JulesEllepi I made a pull request, (sorry for the delay, I've been a bit busy)

**Comment by Staphylococcus** [comment]
Pull request #25 merged

**Comment by mpalian** [comment]
I agree entirely with this objection. This is a draft Constitution for Europe, it makes sense to start with the principles of the Council of Europe https://www.coe.int/en/web/portal ... These have been debated for a long time and there are actual institutions behind them. The Council of Europe is unavoidable, as this constitution will also have to adhere to its principles.

**Comment by Mathew-Iron** [comment]
@mpalian, Yes, the principles of the European Council should be taken into account, but all the treaties signed are already adopted within the European Union, so I don't know if it would be really useful to complicate the constitution. The constitution here is being created with the aim of giving a simpler, clearer guideline to all member states, although we've only “finished” the part on freedoms (which was created just in case of doubt to preserve fundamental freedoms independently of the other treaties signed) that's not the only reason for this constitution's existence. It will also describe the functioning of the government, its institutions, the whole voting system, how it works...
